common/tests/CMakeLists.txt

```diff
@@ -29,5 +31,14 @@ if(${run_int_tests})
         build_test_folder(gballoc_ll_mimalloc_int)
         build_test_folder(gballoc_ll_win32heap_int) 
         build_test_folder(gballoc_ll_passthrough_int)
+        
+        #this builds and runs the same test for gballoc_hl but with all the implementations of the underlying gballoc_ll.
+        #this scripts sets a variable (GBALOC_LL_IMPL) that is picked as underlying implementatio of gballoc_ll and consequentially as file name (gballoc_ll_${GBALOC_LL_IMPL})
+        foreach(GBALOC_LL_IMPL ${GBALLOC_LL_TYPE_VALUES})
+        #set(GBALOC_LL_IMPL "PASSTHROUGH")
+            STRING(TOLOWER ${GBALOC_LL_IMPL} gballoc_ll_impl_lower)
+            build_test_folder(gballoc_hl_passthrough_int BINARY_DIR _${gballoc_ll_impl_lower})

```

Comment:
>_ [](start = 68, length = 1)

leading underscore in directory name intentional?

Reply:
yeah, I was not aware that it shouldn't be done. Results in a folder such as this: "common\tests\gballoc_hl_passthrough_int\_mimalloc\dll".


---
In reply to: [458466489](https://github.com/Azure/azure-c-pal/pull/12#discussion_r458466489) [](ancestors = 458466489)

Reply:
Maybe I can do better - moment. :)

---
In reply to: [458501872](https://github.com/Azure/azure-c-pal/pull/12#discussion_r458501872) [](ancestors = 458501872,458466489)

Reply:
alright, changed CMakeLists.txt to add an "_" be convention in some other place.  NOw folders on the drive look like: "tests\gballoc_hl_passthrough_int\mimalloc\dll"

---
In reply to: [458501872](https://github.com/Azure/azure-c-pal/pull/12#discussion_r458501872) [](ancestors = 458501872,458466489)